server/context.go

```diff
@@ -141,6 +141,11 @@ func NewContext() *Context {
 	return ctx
 }
 
+// Insecure indicates whether or not SSL is to be used.
+func (ctx *Context) Insecure() bool {
```

Comment:
Insecure should say purely in the base/context rather than be a combination of flags.
Instead, `--dev` should set the default value of `--insecure` to true in the flag initializer.
I should still be able to pass `--dev` and `--insecure=false` if I want to play with certs.